packages/grid/test/keyboard-navigation.test.js

```diff
@@ -1889,6 +1890,45 @@ describe('keyboard navigation', () => {
 
       grid.removeEventListener('cell-focus', spy);
     });
+
+    // Separate test suite for Chrome, where we use a workaround to dispatch
+    // cell-focus on mouse up
+    describe('chrome', () => {
```

Comment:
```suggestion
    (isChrome ? describe : describe.skip)('chrome', () => {
```

Reply:
Done, also removed the individual test skips.